cubi_isa_templates/isatab-cancer/{{cookiecutter.__output_dir}}/a_{{cookiecutter.s_file_name}}_transcriptome_profiling.txt

```diff
@@ -1,5 +1,6 @@
 Sample Name	Protocol REF	Parameter Value[Concentration measurement]	Performer	Date	Extract Name	Characteristics[Concentration]	Unit	Term Source REF	Term Accession Number	Protocol REF	Parameter Value[Provider name]	Parameter Value[Provider contact]	Parameter Value[Provider project ID]	Parameter Value[Provider sample ID]	Parameter Value[Provider QC status]	Parameter Value[Requestor contact]	Parameter Value[Requestor project]	Parameter Value[Requestor sample ID]	Parameter Value[Concentration measurement]	Parameter Value[Library source]	Parameter Value[Library strategy]	Parameter Value[Library selection]	Parameter Value[Library layout]	Parameter Value[Library kit]	Parameter Value[Library strand specificity]	Comment[Library kit catalogue ID]	Parameter Value[Target insert size]	Parameter Value[Wet-lab insert size]	Parameter Value[Barcode kit]	Parameter Value[Barcode kit catalogue ID]	Parameter Value[Barcode name]	Parameter Value[Barcode sequence]	Performer	Date	Extract Name	Characteristics[Folder name]	Characteristics[Concentration]	Unit	Term Source REF	Term Accession Number	Protocol REF	Parameter Value[Platform]	Parameter Value[Instrument model]	Parameter Value[Target read length]	Parameter Value[Target read length (mate 2)]	Parameter Value[Base quality encoding]	Parameter Value[Center name]	Parameter Value[Center contact]	Performer	Date	Raw Data File
-{% set sample_names = cookiecutter.sample_names.split(",") -%}																																																			
-{%- for sample_name in sample_names -%}																																																			
-{{sample_name|replace("-", "_")}}-N1	Nucleic acid extraction RNA-Seq				{{sample_name|replace("-", "_")}}-N1-RNA1					Library construction RNA-Seq										TRANSCRIPTOMIC	RNA-Seq	PolyA	PAIRED	Illumina TruSeq Stranded mRNA	REVERSE										{{sample_name|replace("-", "_")}}-N1-RNA1-RNA-Seq1	{{sample_name}}					Nucleic acid sequencing RNA-Seq	ILLUMINA	Illumina HiSeq 4000			Phred+33					
+{% set source_names = cookiecutter.source_names.split(",") -%}																																																			
+{%- for source_name in source_names -%}																																																			
+{{source_name|replace("-", "_")}}-N1	Nucleic acid extraction RNA-Seq				{{source_name|replace("-", "_")}}-N1-RNA1					Library construction RNA-Seq										TRANSCRIPTOMIC	RNA-Seq	PolyA	PAIRED	Illumina TruSeq Stranded mRNA	REVERSE										{{source_name|replace("-", "_")}}-N1-RNA1-RNA-Seq1	{{source_name}}					Nucleic acid sequencing RNA-Seq	ILLUMINA	Illumina HiSeq 4000			Phred+33					
```

Comment:
Is there a reason why the `Library selection`, `Library layout`, `Library kit`, `Library strand specificity` & `Instrument model` have all pre-defined values.

This is different from the exome data, which uses cookiecutter variables for those quantities.

Reply:
This is sth inherited from the original tumor_normal templates which we definitely should discuss on Friday.